### PR TITLE
remove Directory.Build.targets

### PR DESCRIPTION
**Pull request by bruno-garcia**
This branch will be used to release the Unity SDK for the time being

### CODE REVIEW

File: src/Sentry/Envelopes/Envelope.cs [hunk 1]
```diff
@@ -132,12 +132,7 @@ public async Task SerializeAsync(Stream stream, IDiagnosticLogger? logger, Cance
                     }
                     catch (Exception exception)
                     {
-                        if (logger is null)
-                        {
-                            throw;
-                        }
-
-                        logger.LogError("Failed to add attachment: {0}.", exception, attachment.FileName);
+                        logger?.LogError("Failed to add attachment: {0}.", exception, attachment.FileName);
```
Comment (lucas-zimerman) [inline src/Sentry/Envelopes/Envelope.cs:135]: Might be too picky but shouldn't this be part of a separate PR? That way we could merge without Unity blocking the Directory.Build.Targets change.

**Comment by SimonCropp** [comment]
@bruno-garcia can this die?

**Comment by bruno-garcia** [comment]
Yes! Thanks Simon
